core/build.gradle.kts

```
@@ -113,8 +113,6 @@ inner class GitInfo {
     val repository: String
 
     init {
-        // On Jenkins, a detached head is checked out, so indra cannot determine the branch.
-        // Fortunately, this environment variable is available.
         branch = indraGit.branchName() ?: System.getenv("BRANCH_NAME") ?: "DEV"
```

```suggestion
        branch = indraGit.branchName() ?: "DEV"
```
`BRANCH_NAME` is a Jenkins env var